server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -272,6 +274,58 @@ public void testIndexReopenClose() throws Exception {
         verifyStoreContent();
     }
 
+    public void testScrollWithConcurrentIndexAndSearch() throws Exception {
+        final String primary = internalCluster().startDataOnlyNode();
```

Comment:
Sorry didn't realize this on first pass - remote store code is not invoking the new store method yet until https://github.com/opensearch-project/OpenSearch/issues/7802 is done.  I think we need to skip this for remote store case or we will see flaky failures.

Reply:
Thanks @mch2 for pointing this out. Updated test.